tests/test_codetiming.py

```diff
@@ -10,6 +10,8 @@
 
 # Codetiming imports
 from codetiming import Timer, TimerError
+import time
+from math import isnan
```

Comment:
Just a small import nitpick: `math` and `time` are in the standard library so they should be imported in that section, sorted properly :blush:

Reply:
sorry, done now :) they now comply with `isort` here and in the _timer module

Reply:
Ah... thanks for picking up the slack in `_timer` as well :+1: